dim/manage_dim

```diff
@@ -53,16 +53,18 @@ def update_validity():
         dim.models.db.session.commit()
 
 
-@manager.option('-n', '--dry-run', dest='dry_run', action='store_true')
-def ldap_sync(dry_run):
+@manager.option('-n', '--dry-run', dest='dryrun', action='store_true')
+@manager.option('-f', '--ignore-deletion-threshold', dest='ignore_deletion_threshold', action='store_true')
+def ldap_sync(dryrun, ignore_deletion_threshold):
     '''Update Users, Group, and Departments from LDAP'''
-    dim.ldap_sync.ldap_sync(dry_run=dry_run)
+    dim.ldap_sync.ldap_sync(dryrun=dryrun, ignore_deletion_threshold=ignore_deletion_threshold)
```

Comment:
The option dryrun doesn't exist in the ldap_sync function. Only ignore_deletion_treshhold exists. Is that on purpose?

Reply:
Yes, the `ldap_sync` function is being wrapped by `@transaction`, which has the `dryrun` parameter.
https://github.com/1and1/dim/blob/634b09a325a49c65bcd218709b57fa5c20e2539c/dim/dim/transaction.py#L91-L107

Reply:
Okay, thank you :)